packages/asc-ui/src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { HTMLAttributes } from 'react'
+import ErrorMessageStyle from './ErrorMessageStyle'
+
+export interface ErrorMessageProps {
+  message: string
+}
+
+const ErrorMessage: React.FC<
+  ErrorMessageProps & HTMLAttributes<HTMLDivElement>
+> = ({ message, ...otherProps }) => {
+  return (
+    <ErrorMessageStyle role="alert" data-testid="root" {...otherProps}>
```

Comment:
Why is the data-testid called `root`? How about `errorMessage`? So it will have more meaning when used in the implementations.

Reply:
Good catch, we're not actually using this test id anymore so I removed it.